### PR TITLE
[Revamp Shipping Labels] Customs Form Part 1

### PR DESCRIPTION
**Pull request by toupper**
Part of: #13784 

## Description

With this PR we implement the first part of the Customs screen UI. The Products Details UI and the business logic will come in subsequent PRs.

## Steps to reproduce

1. Go to orders
2. Tap on a completed order eligible for shipping labels purchase
3. Tap on Create Shipping Labels
4. Tap on the Customs Row
5. See that the UI is there
( See video)

## Screenshots

https://github.com/user-attachments/assets/d0fe3835-05c0-4fab-befd-7a92622cca4c

---
- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [ ] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [ ] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [ ] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on all devices (phone/tablet) and no regressions are added.

### CODE REVIEW

File: WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsFormViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+
+final class WooShippingCustomsFormViewModel: ObservableObject {
+    @Published var internationalTransactionNumber: String
+    @Published var returnToSenderIfNotDelivered: Bool
+
+    let contentType: WooShippingContentType = .merchandise
+    let restrictionType: WooShippingRestrictionType = .none
+
+    let itnInfoURL = URL(string: "https://pe.usps.com/text/imm/immc5_010.htm")
```
Comment (bozidarsevo) [inline WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsFormViewModel.swift:10]: I see that we have this url in 2 more places, could we maybe put it to some URL extension or some other place where we would have it defined once and referenced in places where we need it?

File: WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsRow.swift [hunk 2]
```diff
@@ -27,16 +28,22 @@ struct WooShippingCustoms: View {
...
+                showCustomsForm.toggle()
             }
             .accessibilityLabel(Text(Localization.editButtonAccessibilityLabel))
         }
         .padding(Layout.borderPadding)
         .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderWidth)
+        .sheet(isPresented: $showCustomsForm) {
+            WooShippingCustomsForm(viewModel: WooShippingCustomsFormViewModel(internationalTransactionNumber: "123",
+                                                                              returnToSenderIfNotDelivered: true))
+
```
Comment (bozidarsevo) [inline WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsRow.swift]: nit: extra new line

File: WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsForm.swift [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    @Environment(\.presentationMode) var presentationMode
+    @ObservedObject var viewModel: WooShippingCustomsFormViewModel
+    @State private var isShowingITNInfoWebView = false
+
+    private var contentTypeSelectionView: some View {
+        Menu {
+            // show selection
+            ForEach(WooShippingContentType.allCases, id: \.self) { option in
+                Button {
+                } label: {
```
Comment (bozidarsevo) [inline WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsForm.swift:14]: Button action for this Menu and Menu below will be done separately?

Comment (bozidarsevo) [reply WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsForm.swift:14]: I suppose you plan to just change those `let`s in `WooShippingCustomsFormViewModel` to `var` and have it like this:
`viewModel.contentType = option`?

**Comment by bozidarsevo** [comment]
> Closes: #13784

You can change this to "Part of " because if it says Closes when we merge this PR it will close the issue alltogether.

**Comment by toupper** [comment]
> You can change this to "Part of " because if it says Closes when we merge this PR it will close the issue altogether.

True! It was some reflex action. Thanks for the reminder

**Comment by bozidarsevo** [comment]
> True! It was some reflex action. Thanks for the reminder

I will update the github template to have Close/Part/*: # for easier use and reminder to ourselves

File: WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsForm.swift (hunk 3 above)
Comment (toupper) [reply WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsForm.swift:14]: Exactly, I will do that when implementing the Business Logic.

File: WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsRow.swift (hunk 2 above)
Comment (toupper) [reply WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsRow.swift]: Thanks! Removed in 974cce396fb6e487045a6ab08cb21cc0a1aa80ea

**Review by bozidarsevo — APPROVED** [review]

**Pull request merged**